test/integration/NodeConnection.spec.ts

```diff
@@ -0,0 +1,121 @@
+import 'mocha'
+import {
+    RadixIdentityManager,
+    RadixIdentity,
+    RadixNodeConnection,
+    RadixLogger,
+    radixUniverse,
+    RadixUniverseConfig,
+    RadixNodeDiscoveryHardcoded,
+    RadixUniverse,
+    RadixBootstrapConfig,
+} from '../../src'
+import { expect } from 'chai'
+import local from '../../src/modules/universe/configs/local.json'
+
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+const timeout = 10000
+let universe: RadixUniverse
+
+describe('BS-306: Check hid on connection', function () {
+    this.timeout(timeout)
+    let nodeConnection: RadixNodeConnection
+
+    before(() => {
+        RadixLogger.setLevel('info')
```

Comment:
This should be set to 'error' to not pollute the logs, changed only for debugging locally